requirements_test.txt

```
@@ -1,6 +1,5 @@
 black==19.10b0
```

We'll need to bump black to support the latest Python versions.
```suggestion
black==22.1.0
```

---

This is only necessary to support newer Python syntax, but given that we're keeping Python3.6 support, the previous version should be fine too.